MethodsHomework/MethodsHomework.cs

```diff
@@ -166,10 +170,116 @@ public class YourAnswers
     {
         //READ ME
         // You write all your methods here.
+
+        public static void DoNothing() { }
         
-    }
+        public static int ReturnIntWithValue25() { return 25; }
+
+        public static int AddTwoNumbers(int x, int y)
+        {
+            return x + y;
+        }
+
+        public static int IncrementMe(int valueToBeIncremented)
+        {
+            int Incremented = ++valueToBeIncremented;
+            return Incremented;
+        }
+
+        public static int DecrementMe(int valueToBeDecremented)
+        {
+            int Decremented = --valueToBeDecremented;
+            return Decremented;
+        }
+
+        public static int CastDoubleToInt(double valueToBeCasted)
+        {
+            int casted = ((int) (valueToBeCasted));
+            return casted;
+        }
+
+        public static double ConvertIntToDouble(int valueToBeConverted)
+        {
+            return Convert.ToDouble(valueToBeConverted);
+        }
+
+        public static string CastMeIntoAString(double valueToBeCastedIntoString)
+        {
+            string castedString = valueToBeCastedIntoString.ToString();
+            return castedString;
+        }
+
+        public static double ParseMeIntoADouble(string valueToBeParsed)
+        {
+            double parsed = Convert.ToDouble(valueToBeParsed);
+            return parsed;
+        }
+
+        public static int ReturnMaxIntValue(int max = 2147483647)
```

Comment:
The maximum `int` value shouldn't depend on any parameters - it just _is_. Also, there's a symbol for this value, so you don't have to type in the number.